services/github/test/handler_utils_test.js

```diff
@@ -106,4 +106,22 @@ helper.secrets.mockSuite(testing.suiteName(), [], function() {
       repo: 'tc',
     }, gc.getRerequestPayload());
   });
+
+  test('Get Time Difference', function() {
+
+    const START_TIMESTAMP = new Date('2024-07-16T18:23:18.118Z');
+    const END_TIMESTAMP_MILLISECONDS = new Date('2024-07-16T18:23:18.128Z');
+    const END_TIMESTAMP_SECONDS = new Date('2024-07-16T18:23:28.118Z');
+    const END_TIMESTAMP_MINUTES = new Date('2024-07-16T18:33:18.118Z');
+    const END_TIMESTAMP_HOURS = new Date('2024-07-17T04:23:18.118Z');
+    const END_TIMESTAMP_DAYS = new Date('2024-07-26T18:23:18.118Z');
+    const END_TIMESTAMP_GENERIC = new Date('2024-07-27T04:33:28.128Z');
+
+    assert.equal(getTimeDifference(START_TIMESTAMP, END_TIMESTAMP_MILLISECONDS), "10 milliseconds");
```

Comment:
this is great! 
would be good to include tests for the missing/invalid dates also since `getTimeDifference` is not validating the input

Reply:
can we receive invalid dates, or is it just to make the tests robust?

Reply:
Someone might always use this function with some invalid payload. And because we don't have strict types here, we could at least add some defensive mechanisms to return `n/a` if something is not a valid date